src/utils/isNewValueForObjectPaths.ts

```diff
@@ -1,6 +1,6 @@
 import { pick } from 'lodash';
 import isEqual from 'react-fast-compare';
 
-export default function isNewValueForObjectPaths(a, b, paths) {
+export default function isNewValueForObjectPaths(a: any, b: any, paths: any) {
```

Comment:
`any` types, where I think `paths` can be `string | string[]` based on Lodash and `a` and `b` are objects. This is the same as for `isNewValueForPath`.